Add an article for the SR2021 knockouts (#272)
srobo/website: _posts/2021-05-01-sr2021-knockouts.md

modules that presented new and different challenges to the competitors. In
addition to the formal league sessions, there were also some more casual
friendlies sessions which allowed the teams to experiment with different
strategies and test their code against other teams’ in a more relaxed
environment.

While this year’s competition presented a different challenge to the competitors
that previous years’, the core challenge to create an autonomous robot remained
the same.

{% include figure.html
           class="right"
           link_to_image="true"
           src="/images/content/news/sr2021/league-4-top-teams.png"
           caption="Teams’ game points per league session." %}

During the league sessions there were some teams which showed strong early
performances. Initially the teams from Hills Road Sixth Form College, looked to
be the teams to beat. The later leagues however were the battleground of a
different collection of teams, with KEGS Chelmsford (KEG), Gymnasium Markt
Indersdorf (MAI) as well as rookie teams St Paul’s College (SPA) and We Robot
(WER) in particular performing the best in the fourth and most complex of the
leauge sessions.

> The Final

Author: Thoughts on extending this into a fuller summary of the knockouts?

Reviewer: Is this now covered by the above "Knockouts" section?

Author: Yes.

up later in the year. Get a team together and start talking to your teachers now!

Results
-------

{% include figure.html
           class="right"
           src="/images/content/news/sr2021/trophy-render.png"
           alt="A podium, with rendered trophies, featuring SPA in Gold, HRS2 in Silver, and MCK in Bronze"
           caption="The podium positions" %}

|        Prize          |            Team
|-----------------------|-----------------------------------------------
| 1<sup>st</sup> Place  | [SPA: St Pauls College, Adelaide](https://www.stpauls.sa.edu.au/)
| 2<sup>nd</sup> Place  | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 3
| 3<sup>rd</sup> Place  | [The Malay College Kuala Kangsar](https://mckk.edu.my)
| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 2
| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [We Robot](https://www.werobot.fr)

> | <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.yshhb.org.bn/)

Reviewer: Turns out we were right before...

```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.secondary-yayasan.edu.bn/en/)
```

Author: :disappointed:

Author: e5bd53cbb7929afba78952e62da5b61b5fdf4b6c

> | <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.secondary-yayasan.edu.bn/en/)

Reviewer: Use link sourced from team

```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.yshhb.org.bn/)
```

Author: For some reason GitHub wouldn't let me provide this via the UI: 8d4c2bf

the same.

{% include figure.html
           class="right"
           link_to_image="true"
           src="/images/content/news/sr2021/league-4-top-teams.png"
           caption="Teams’ game points per league session." %}

During the league sessions there were some teams which showed strong early
performances. Initially the teams from Hills Road Sixth Form College, looked to
be the teams to beat. The later leagues however were the battleground of a
different collection of teams, with KEGS Chelmsford (KEG), Gymnasium Markt
Indersdorf (MAI) as well as rookie teams St Paul’s College (SPA) and We Robot
(WER) in particular performing the best in the fourth and most complex of the
leauge sessions.

The Final
---------

The final match saw SPA and HRS3 battle for the top spot. Both teams having
shown their incredible abilities throughout the competition. Whilst the robots
started off with very different tactics, they were both accruing points very
quickly. Due to a large number of link breakages, the points and leader switched
massively throughout the match, but when the clock struck zero, it was 36 points
to 8 for St Pauls, giving them the title.

> <!-- Video of Final -->
> TODO: make this embed not be the whole stream.
> <iframe
>   class="center video"
>   src="https://www.youtube-nocookie.com/embed/gxJM6ernMqo"

Reviewer: Match cut is live https://www.youtube.com/watch?v=Jwk2kUp02lk

```suggestion
<!-- Video of Final -->
<iframe
  class="center video"
  src="https://www.youtube-nocookie.com/embed/Jwk2kUp02lk"
```

starting zone or to territories already owned could be claimed. At the end of
the match, teams were awarded points based on the territories they owned. Full
details, including the prizes available this year, are available in the
[rulebook][rules].

The Leagues
-----------

Our competition event this year spread the league stage over four weekends
throughout the year. After each league session the game gained additional
modules that presented new and different challenges to the competitors. In
addition to the formal league sessions, there were also some more casual
friendlies sessions which allowed the teams to experiment with different
strategies and test their code against other teams’ in a more relaxed
environment.

While this year’s competition presented a different challenge to the competitors
that previous years’, the core challenge to create an autonomous robot remained
the same.

{% include figure.html
           class="right"
           link_to_image="true"
           src="/images/content/news/sr2021/league-4-top-teams.png"
           caption="Teams’ game points per league session." %}

During the league sessions there were some teams which showed strong early

> performances. Initially the teams from Hills Road Sixth Form College, looked to

Reviewer: ```suggestion
performances. Initially the teams from Hills Road Sixth Form College looked to
```

Reviewer: Probably worth putting their TLA in too, to follow the pattern below

```suggestion
performances. Initially the teams from Hills Road Sixth Form College (HRS1-4) looked to
```

Author: Comma removed. I'm not sure the TLAs adds much here?

in separate files, with good comments. Their code made good use of a state
machine as part of its main control loop and PID control for their movement.

We also encourage teams to share their progress towards their robots throughout
the year. Team YSH from Yayasan Sultan Haji Hassanal Bolkiah earned the Online
Presence Award for their [Instagram posting][ysh-instagram] throughout the year
covering their approach to the competition, strategy and reviews of their
performance in the leagues.

For full details on all the awards, please see the [rulebook][rulebook].

You can see a breakdown of scores for each match, as well as the overall league
ranking on the [competition website](/comp/).

Rewatch the streams
-------------------

If you’d like to relive the highs and lows of the competition livestreams, the
videos remain available on YouTube, as well as a cut-down video of the final:

- [League Session 1](https://www.youtube.com/watch?v=cAvk-nfTUis)
- [League Session 2](https://www.youtube.com/watch?v=RwW5Oz30gbE)
- [League Session 3](https://www.youtube.com/watch?v=MpuhtW4mCKM)
- [League Session 4](https://www.youtube.com/watch?v=E9gF-GZbf5M)

> - [Knockouts and Final](https://www.youtube.com/watch?v=gxJM6ernMqo)

Reviewer: Given this is a post about the knockouts and finals, should we embed this as an actual player?

Author: We already have one embed here and this article is pretty long (as is the livestream). I think we're fine to just leave this as a link.
